storage/pebble/registers.go

```diff
@@ -1,16 +1,20 @@
 package pebble
 
 import (
+	"encoding/binary"
 	"fmt"
 
 	"github.com/cockroachdb/pebble"
+	"github.com/onflow/flow-go/storage"
 
 	"github.com/onflow/flow-go/model/flow"
 )
 
 // library that implements pebble storage for registers
 type Registers struct {
-	db *pebble.DB
+	db           *pebble.DB
+	firstHeight  uint64
+	latestHeight uint64
```

Comment:
these will need to be loaded from the db at some point before the object is ready for use